sc_cost_meter/utils.py

```diff
@@ -76,50 +74,16 @@ def get_marketplace_customer_id(synapse_id):
       },
       TableName=ddb_marketplace_table_name,
       ConsistentRead=True,
-      AttributesToGet=[
```

Comment:
I double checked that if this is omitted then all attributes are returned https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/LegacyConditionalParameters.AttributesToGet.html